addOns/fuzz/src/main/javahelp/org/zaproxy/zap/extension/fuzz/resources/help/contents/dialogue.html

```diff
@@ -25,6 +25,26 @@ <H3>Fuzz Locations tab</H3>
 <H3>Options tab</H3>
 This tab allows you to configure the options to be used when fuzzing.
 
+<H4>Retries on IO Error</H41>
+The number of retries when an input/output error occurs sending a request to the target.
+
+<H4>Max. Errors Allowed</H4>
+If the number of errors exceed this limit, fuzzer will stop its execution.
+
+<H4>Payload Replacement Strategy</H4>
+Rules defined to control the order that multiple payload lists are iterated.
+
+<H4>Concurrent Scanning Threads per Scan</H4>
+The number of threads the fuzzer will use per scan.<br>
+Increasing the number of threads will speed up the scan but may put extra strain on the computer ZAP is running on as well as the target.
+
+<H4>Delay when Fuzzing (in milliseconds)</H4>
+The number of milliseconds between requests by the fuzzer to the target host, usually done to
+avoid getting blocked by the target or if the target implements some sort of throttling requirement.
```

Comment:
It would be better to link to main options page instead of duplicating here again.